foundation/foundation-mda/src/main/resources/profiles.json

```diff
@@ -866,37 +866,6 @@
 			}
 		]
 	},
-	{
```

Comment:
I: No longer need Jenkins deployment templates as it's part of infrastructure chart. This chart was never officially released, so no harm in deleting the foundation-mda profile and targets.